experiment/gerrit-onboarder/onboarder.go

```diff
@@ -373,12 +374,57 @@ func fetchMetaConfig(host, repo, branch string) error {
 	return nil
 }
 
+func createCR() error {
+	commitMessage := fmt.Sprintf("Grant the Prow cluster read and label permissions\n\nChange-Id: I%s", bumper.GitHash(fmt.Sprintf("%d", rand.Int())))
+	if err := bumper.Call(os.Stdout, os.Stderr, "git", "commit", "-a", "-v", "-m", commitMessage); err != nil {
+		return fmt.Errorf("unable to commit: %v", err)
+	}
+	if err := bumper.Call(os.Stdout, os.Stderr, "git", "push", "origin", "HEAD:refs/for/refs/meta/config"); err != nil {
+		return fmt.Errorf("unable to push: %v", err)
+	}
+	return nil
+}
+
+func getDiff() (string, error) {
+	var diffBuf bytes.Buffer
+	var errBuf bytes.Buffer
+	if err := bumper.Call(&diffBuf, &errBuf, "git", "diff"); err != nil {
+		return "", fmt.Errorf("diffing previous bump: %v -- %s", err, errBuf.String())
+	}
+	return diffBuf.String(), nil
+}
+
 func main() {
 	o, err := parseAndValidateOptions()
 	if err != nil {
 		logrus.Fatal(err)
 	}
 
+	curDir, err := os.Getwd()
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	workDir, err := ioutil.TempDir("", "gerrit_onboarding")
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	defer os.RemoveAll(workDir)
+
+	err = os.Chdir(workDir)
```

Comment:
This looks bash to me, can we not using `chDir`? If it's for shelling out to git, `exec` package does support `Dir` https://golang.org/pkg/os/exec/#Cmd

Reply:
Created a execInDir function and replaced all uses of Call with this to avoid having to chDir.